scripts/linters/pre_commit_linter.py

```diff
@@ -314,7 +314,7 @@ def _get_file_extensions(file_extensions_to_lint):
 
     Args:
         file_extensions_to_lint: list(str). The list of file extensions to be
-        linted checked.
+            linted checked.
```

Comment:
linted checked --> lint-checked (or "linted and checked")

Reply:
Done